lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -1011,22 +1011,20 @@ export async function updatePr({
       };
     }
 
-    const { body: updatedPr } = await bitbucketServerHttp.putJson<{
-      version: number;
-      state: string;
-    }>(
+    const { body: updatedPr } = await bitbucketServerHttp.putJson<BbsRestPr>(
```

Comment:
```suggestion
    const { body: updatedPr } = await bitbucketServerHttp.putJson<BbsRestPr & {
        version: number;
      }>(
```

wonder why `version` is optional. 🤔  

we should switch to zod schema as soon as possible

Reply:
its optional in the `BbsRestPr` interface..hence :) Should be changed, is it okay to do in this pr?